specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2021-11-01-preview/common.json

```diff
@@ -0,0 +1,91 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-11-01-preview",
+    "title": "KeyVaultManagementClient",
+    "description": "The Azure management API provides a RESTful set of web services that interact with Azure Key Vault."
+  },
+  "paths": {},
+  "definitions": {
```

Comment:
Looking at the aforementioned diff, there's a bunch of definitions missing like `Sku`. Were those just unused? Given there's no major errors in validation I'll assume so, but wanted to bring it up just to check.

Reply:
My guess is that they are not used. I just copied the latest api and added the new api-version and excluded secure key release and auto key rollover for the stable version